app/src/main/java/openfoodfacts/github/scrachx/openfood/features/product/view/summary/SummaryProductFragment.kt

```diff
@@ -462,6 +474,44 @@ class SummaryProductFragment : BaseFragment(), ISummaryProductPresenter.View {
         }
     }
 
+    private fun showListChips() {
+
+        // remove the existing childviews on chip group if any
+        binding.listChips.removeAllViews()
+
+        val listDao: YourListedProductDao = OFFApplication.daoSession.yourListedProductDao
```

Comment:
Is it useful to have this explicit type?

Reply:
No. I removed it.